### PR TITLE
Add new CAS & Migration path

### PR DESCRIPTION
**Pull request by pvizeli**

### CODE REVIEW

File: helpers/codenotary/action.yml [hunk 1]
```diff
@@ -3,30 +3,50 @@ description: 'GitHub action helper: CodeNotary'
...
       with:
         vcn_version: ${{ inputs.vcn_version }}
 
+    - uses: ./helpers/cas
+      with:
+        version: ${{ inputs.version }}
+
     - shell: bash
+      name: CodeNotary depricated VCN
+      if: ${{ input.user != "" && input.password != "" && input.organisation != "" }}
```
Comment (ludeeus) [inline helpers/codenotary/action.yml:49]: Steps in `composite` actions do not support `if` definitions, that is why we have those under the `run` definitions.

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: helpers/codenotary/action.yml (hunk 1 above)
Comment (pvizeli) [reply helpers/codenotary/action.yml:49]: that suck, but okay will migrate it :D

File: helpers/codenotary/action.yml [hunk 2]
```diff
@@ -3,31 +3,54 @@ description: 'GitHub action helper: CodeNotary'
...
         vcn_version: ${{ inputs.vcn_version }}
 
+    - uses: ./helpers/cas
+      with:
+        version: ${{ inputs.version }}
+
     - shell: bash
+      name: CodeNotary depricated VCN
       run: |
+        if [[ -z "${{ input.user }}" || -z "${{ input.password" }} || -z "${{ input.organisation }}" ]]; then
```
Comment (ludeeus) [inline helpers/codenotary/action.yml:50]:
```suggestion
        if [[ -z "${{ inputs.user }}" || -z "${{ inputs.password" }} || -z "${{ inputs.organisation }}" ]]; then
```

File: helpers/codenotary/action.yml [hunk 3]
```diff
@@ -40,3 +63,20 @@ runs:
         VCN_USER: ${{ inputs.user }}
         VCN_PASSWORD: ${{ inputs.password }}
         VCN_NOTARIZATION_PASSWORD: ${{ inputs.password }}
+
+    - shell: bash
+      name: CodeNotary Open Source Attestation Service
+      run: |
+        if [[ -z "${{ input.signer }}" || -z "${{ input.token" }} ]]; then
```
Comment (ludeeus) [inline helpers/codenotary/action.yml:70]:
```suggestion
        if [[ -z "${{ inputs.signer }}" || -z "${{ inputs.token" }} ]]; then
```

File: helpers/codenotary/action.yml [hunk 4]
```diff
@@ -40,3 +63,20 @@ runs:
...
+
+        if ! cas authenticate --silent ${{ inputs.source }}; then
+          cas login
+          cas notarize --public ${{ inputs.source }}
+        else
+          echo "Target is already verified."
+        fi
+      env:
+        CAS_SIGNERID: ${{ input.signer }}
+        CAS_API_KEY: ${{ input.token }}
```
Comment (ludeeus) [inline helpers/codenotary/action.yml:82]:
```suggestion
        CAS_SIGNERID: ${{ inputs.signer }}
        CAS_API_KEY: ${{ inputs.token }}
```

**Review by ludeeus — APPROVED** [review]

**Pull request merged**